sfyra/pkg/tests/server.go

```diff
@@ -259,6 +259,127 @@ func TestServerAcceptance(ctx context.Context, metalClient client.Client, vmSet
 	}
 }
 
+// TestServerPaused makes sure the paused bool works.
+func TestServerPaused(ctx context.Context, metalClient client.Client, vmSet *vm.Set) TestFunc {
+	return func(t *testing.T) {
+		const numDummies = 3
+
+		// create dummy servers to test with
+		dummySpec := v1alpha1.ServerSpec{
+			CPU: &v1alpha1.CPUInformation{
+				Manufacturer: "DummyManufacturer",
+			},
+		}
+
+		for i := 0; i < numDummies; i++ {
+			serverName := fmt.Sprintf("dummyserver-%s", strconv.Itoa(i))
+			_, err := createDummyServer(ctx, metalClient, serverName, dummySpec)
+			require.NoError(t, err)
+		}
+
+		dummyServers := &v1alpha1.ServerList{}
+
+		labelSelector, err := labels.Parse("dummy-server=")
+		require.NoError(t, err)
+
+		// wait for all the servers to be registered
+		require.NoError(t, retry.Constant(1*time.Minute, retry.WithUnits(10*time.Second)).Retry(func() error {
```

Comment:
I don't think there's a way dummyServers can be not visible immediately after creation?

Reply:
fixed